tasks/ceph.py

```diff
@@ -558,7 +558,19 @@ def cluster(ctx, config):
                         stdout=StringIO(),
                         )
 
-                remote.run(args= ['yes', run.Raw('|')] + ['sudo'] + mkfs + [dev])
+                try:
+                    remote.run(
+                        args=['yes', run.Raw('|')] + ['sudo'] + mkfs + [dev]
+                    )
+                except run.CommandFailedError:
```

Comment:
Don't you want to be a little more specific about this? Like only trying again if it's btrfs, and making sure below that we're removing the "-f" and not something else?
(Somebody someday is going to change that arguments list and they might put it at the front instead of the end.)